### ISSUE TITLE
navbar example

### ISSUE THREAD

**Issue by joepadmiraal** [kaliber5/ember-bootstrap#48]
Can you please add a navbar example with the collapse component to the documentation?

Thanks,
Joep

**Comment by zoamel** [comment]
+1 For that

**Comment by plainsimple123** [comment]
Wow! What a coincidence I'm also looking for that. +1.

**Comment by plainsimple123** [comment]
Well, after relentless searching I found the example in ember-bootstrap/tests/dummy/app/templates/collapse.hbs. Enjoy @joepadmiraal and @zoamel!

It would be great if that is shown in the published documentation. But nonetheless, great job to kaliber5!

**Comment by joepadmiraal** [comment]
@plainsimple123 Thanks for pointing to that example, but that is not exactly what I am looking for.
I'm looking for an example which includes the navbar.

I currently have something like this, but it is not working correctly.
For example when the navbar is closed in a small window and the window is made larger, the navbar stays invisible.

```
<div class="navbar-header">
  {{#bs-button class="navbar-toggle" toggle=true active=notCollapsed action="toggle"}}
    <span class="sr-only">Toggle navigation</span>
    <span class="icon-bar"></span>
    <span class="icon-bar"></span>
    <span class="icon-bar"></span>
  {{/bs-button}}
  <span class="navbar-brand active">Ember.js example</span>
</div>

{{#bs-collapse collapsed=collapsed}}
  <nav class="navbar-collapse bs-navbar-collapse" role="navigation">
    <ul class="nav navbar-nav">
        {{#active-link}} {{#link-to 'groups'}}Groups{{/link-to}} {{/active-link}}
        {{#active-link}} {{#link-to 'students'}}Students{{/link-to}} {{/active-link}}
    </ul>
  </nav>
{{/bs-collapse}}
```

**Comment by simonihmig** [comment]
@plainsimple123 I plan to integrate the demo app from tests/dummy into the official documentation page (http://kaliber5.github.io/ember-bootstrap/), hopefully soon.

**Comment by simonihmig** [comment]
@joepadmiraal Can you please show me a code example of what you want to achieve using the original bootstrap.js? I am not sure I unterstand what your goal is...

**Comment by joepadmiraal** [comment]
@simonihmig Basicaly its just [this navbar example](http://getbootstrap.com/components/#navbar-default) from the official Bootstrap site.

When the browser window is made smaller the navbar will hide the buttons and show a button to collapse and expand the button list.

**Comment by kwasielew** [comment]
Hello, I would like to add to this topic my question:
Is there any example of ember-bootstrap with some side nav element menu?

**Comment by basz** [comment]
I think joepadmiraal wants an example of how to target the `bs-collapse` component from the `bs-button`.

In bootstrap you would spilt a class via data-target attribute on the button. Off course ember doesn't work that way.

I'm interested in such a solution too...

and ps. very nice library this is... thanks a LOT

**Comment by basz** [comment]
I figured it out;

https://gist.github.com/basz/99772922468152fb946d

**Comment by srvance** [comment]
Nice piece of work, @basz. If I understand your gist, you're using it to collapse a sidebar, not the kind of responsive alternate menu presentation shown in the Bootstrap navbar example that @joepadmiraal cited.

Digging into it, I agree that the cross-component signaling isn't very Ember-y and would require a different approach. I also found that using markup or componentization to group the elements needing collapse breaks ember-bootstrap's strong assumptions about the relationships between child and parent components (as detected by nearestOfType) in a way that will requires some redesign or other accommodation.

**Comment by srvance** [comment]
I've started adding bs-navbar functionality in https://github.com/srvance/ember-bootstrap/commit/7763bf2690d265019bff48c049863bec26faf8c6.

**Comment by simonihmig** [comment]
This has been implemented by #123 and will be released as 0.9.0 soon. All credits go to @srvance!
